packages/orchestrator/test_consensus_gate.py

```
@@ -0,0 +1,435 @@
+
+
+def test_tally_direct():
+    """Exercise tally() without voter mocks."""
+    claim = sample_claim(blast=BlastRadius.MODULE)
+    vs = [
+        Vote(voter="a", vote=1, rationale="ok"),
+        Vote(voter="b", vote=1, rationale="ok"),
+    ]
+    assert tally(claim, vs) == Outcome.APPROVED
+
+    vs_reject = vs + [Vote(voter="c", vote=-1, rationale="no")]
```

Ruff (0.15.9)

[warning] Consider `[*vs, Vote(voter="c", vote=-1, rationale="no")]` instead of concatenation

Replace with `[*vs, Vote(voter="c", vote=-1, rationale="no")]`

(RUF005)